frontend/common/src/helpers/runtimeVariable.ts

```diff
@@ -5,14 +5,28 @@ interface HasServerConfig {
 }
 
 /**
- * A function to retrieve a environment variable value from the window object
+ * Retrieve an environment variable value from the window object
  */
 export const getRuntimeVariable = (name: string): string | undefined => {
   const windowWithConfig = window as unknown as HasServerConfig;
   // eslint-disable-next-line no-underscore-dangle
   return windowWithConfig.__SERVER_CONFIG__?.get(name);
 };
 
+/**
+ * Retrieve an environment variable value from the window object with error if missing
+ */
+export const getRuntimeVariableNotNull = (name: string): string => {
+  const runtimeVariable = getRuntimeVariable(name);
+  if (runtimeVariable) {
+    return runtimeVariable;
+  }
+
+  // eslint-disable-next-line no-console
+  console.error(`Missing mandatory runtime variable: ${name}`);
```

Comment:
Weird, I'm not seeing this error popup due to `const runtimeVariable = getRuntimeVariable(name);` constant returning "(none)" instead of an empty string, which is technically `true`. You would think it would return `undefined`, since it's a Map. Double checking again.

Reply:
![image](https://user-images.githubusercontent.com/22059495/183733091-36384b7b-4ef9-4684-9fcb-56109119ea17.png)

Reply:
Good catch!  I must have tested with admin but the filterEmpty function was missing on ts and iap.  Added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/3575/commits/ac69b7064982860ce7a07a326d36a8dfa6a779c3 .
![image](https://user-images.githubusercontent.com/8978655/183737280-38b533f3-7711-4278-85b1-cb83747c26f7.png)

Reply:
To test the fix you'll need to rerun webpack build to copy the sjs file from `public` to `dist` (or manually copy) and probably rebuild the php container.